style.css

```diff
@@ -0,0 +1,47 @@
+h1 {
+  font-size: 200px;
+  color: white;
+ 
+  
+}
+
+h2 {
+  color: white;
+  text-align: left;
+ 
+   
+
+}
+body {
+  
+  font-family: "Helvetica Neue", sans-serif;
+  font-size: 24px;    
+  
+  
+}
+
+P {
+  font-size: 44px; 
+  text-align: center;
+    color: white
+
+}
+.written-by{
+  color: white;
+    text-align: right;
+}
+  .jac{
+    text-align: right;
+    color: darkblue
+  }
+.link{
+  text-align: right;
+  color: darkblue;
+
+}
+
+img {
+  img-align: right;
```

Comment:
I totally get what you were trying to achieve here, but `img-align` isn't a valid CSS property. We're going to dig more into layout in the coming weeks, so you'll soon know the CSS necessary to do what you were going for here!